pkg/skaffold/parser/config.go

```diff
@@ -73,6 +73,16 @@ func GetAllConfigs(opts config.SkaffoldOptions) ([]*latestV1.SkaffoldConfig, err
 	return cfgs, nil
 }
 
+// GetBase returns the correct base address (work dir) for a given config file
+func GetBase(isDependency bool, cfgFile string) (string, error) {
```

Comment:
based on comment in `runner.go` we won't need to export this function

Reply:
also won't need to commit all the dummy file changes since it won't run on `skaffold generate-pipeline`

Reply:
Changed to no longer be exported.  Also deleted dummy.yaml files under `integration/testdata/generate_pipeline/`